library/src/main/java/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -62,6 +63,10 @@
     RANGE
   }
 
+  // List of that require manually creation of YYYY MMMM date format
```

Comment:
List of **Locales**

Reply:
maybe rename this to `explicitlyNumericYearLocaleLanguages`?

Reply:
I agree, I'll rename it.